src/components/conference-card.tsx

```diff
@@ -0,0 +1,277 @@
+import { useEffect, useState } from "react";
+import type { ReactNode } from "react";
+import type { Event } from "@/content/events/events";
+import type { Talk } from "@/content/talks/talks";
+import type { Speaker } from "@/content/speakers/speaker";
+import type { Sponsor } from "@/content/sponsors/sponsors";
+import { cn, formatTime } from "@/lib/utils";
+import { LocationBadge } from "@/components/location-badge";
+import { LanguageBadge } from "@/components/language-badge";
+import { FavoriteButton } from "@/components/favorite-button";
+import { getEntry } from "astro:content";
+import { FavoritesContextProvider } from "@/context/FavoritesContext";
+
+type ConferenceCardProps = {
+  activity: Event["schedule"][number];
+  event: Event;
+};
+
+type BreakCardProps = {
+  break: Event["schedule"][number];
+  event: Event;
+};
+
+function TimeAndDuration(props: {
+  startTime?: Date | undefined;
+  duration?: number;
+  className?: string;
+  children?: ReactNode;
+}) {
+  return (
+    props.startTime && (
+      <div
+        className={cn(
+          "flex flex-row gap-2 text-sm text-gray-300",
+          props.className,
+        )}
+      >
+        <time dateTime={props.startTime?.toISOString()}>
```

Comment:
```suggestion
        <time dateTime={props.startTime.toISOString()}>
```

The `?` is not needed here, you already made sure that startTime is defined l.31 ;)